xml/vm_security.xml

```diff
@@ -25,22 +33,21 @@
   <sect1 xml:id="vm-security-enable-confidential-compute-module">
```

Comment:
This is about the title of the previous subsection which needs to be sentence-case: _Supported hardware_.
Also, please add "to" to the paragraph below the title: _A system with an AMD EPYC (3rd Gen or newer) is required **to** run AMD SEV-SNP virtual machines_.